src/api/app/models/project.rb

```diff
@@ -1539,6 +1544,15 @@ def branch?
     name.include?(':branches:') # Rather ugly decision finding...
   end
 
+  def categories
```

Comment:
:thought_balloon: next stop, move this to the DB...

Reply:
Ok, adding it to my todo list